_posts/2017-03-16-two-months-of-tech-interviews.md

```diff
@@ -0,0 +1,220 @@
+
+During this time, I made the mistake of scheduling a phone interview before I was fully confident. I failed pretty spectacularly and ruined a really great opportunity... not good 😢.
+
+### Resume cleanup
+
+I updated [my resume](/resume) before starting applications and a few times throughout the process. Some things I tried to do:
+
+- **One page**: Companies get tons of resumes and they have limited time to read each one.
+- **Use bullets**: They are easier to skim and help separate discreet accomplishments.
+- **Show, don't tell**: Let experiences speak for themselves. Building something with a 10/10 rating is better than saying "my project was popular".
+- **Less is more**: At the start I thought more things meant more impressive. Later on I realized that each item has diminishing returns. Too much just creates clutter.
+- **Consistency:** I tried to be consistent with how I mention technologies and structure items.
+
+With each update I shared my resume with a bunch of people for feedback. Here's a .gif of all the iterations it went through.
+
+![Resumes 2017](/assets/posts/resumes-2017.gif){: .center-image }
+
+Resume evolution (6 versions)
+{: .image-caption }
+
+It changed quite a bit over the course of the 2 months. Looking back, I feel a tad gross having sent out the original version 😅.
+
+### Sending applications
+
+Over the course of my 2 month search, I applied to 43 companies.
+
+Here are the stats<sup>\*</sup>:
+
+| Type of application | # Interested | % Interested |
+|:--------------------|:-------------|:-------------|
+| Online 			  | 4 / 22		 | 18.2 %		|
+| Referral 			  | 12 / 21		 | 57.1 %		|
+| **Total**			  | **16 / 43**	 | **37.2 %**	|
+|---------------------|--------------|--------------|
+
+\* *Interested means I was contacted for some type of interview.*
+
+Something to note: **Referrals were 3x more likely to get a response.**
+
+I found referrals via:
+
+- Working with the jobs team at the [Recurse Center](https://www.recurse.com/hire).
+- Contacting past coworkers.
+- Broadcasting my job hunt to friends on Facebook.
+- Joining online tech communities (eg. a slack group for technologists in Toronto).
+- Talking to independent recruiters.
+
+I would highly recommend reaching out to your networks. Referrals are so much more reliable.
+
+Of the places that weren't interested, it generally fell into the category of *"looking for senior engineers"* or *"looking for US based engineers"* (I'm Canadian 🇨🇦).
+
+---  
+***Side note:***  
+*Extra thanks to the Recurse Center. They were incredibly helpful during my job hunt.*
+
+*When I originally applied to RC, I thought the recruiting side was just a necessary evil. Now I see it's an invaluable service.*  
+
+---
+
+### Interviews
+
+Of the 43 companies I applied to, 16 went forward with some type of interview.
+
+The process generally followed:
+
+1. Coding test (sometimes)
+1. Phone interview
```

Comment:
In markdown, you can write lists as 

```
1. Item
1. Item
1. Item
```
And it gets converted to
```
1. Item
2. Item
3. Item
```

It's nice for when you need to rearrange things :)